llvm/test/CodeGen/AArch64/sqrshr-uqrshl-unpredictable.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llc -mtriple armv8.1m.main -mattr=+mve %s -o - | FileCheck %s
```

Comment:
This test should be under CodeGen/Arm or CodeGen/Thumb2, as it's not AArch64.

I would also personally use the update_llc_test_checks script to generate the check lines.

Reply:
Thank you, I have moved the test

Unfortunately I couldn't get the update_llc_test_checks took to work, I ran into the same issue as https://github.com/llvm/llvm-project/issues/66409